teensytoany/teensytoany.py

```diff
@@ -276,7 +276,14 @@ def i2c_write_read(self,
             length=num_bytes, byteorder='big',
             signed=False)
 
-    def i2c_write_payload(self, address: int, payload: Sequence) -> None:
+    def i2c_write_payload(self, address: int, register_address: int, payload: Sequence) -> None:
+
+        data =  ' '.join([f"0x{val:02x}" for val in payload])
```

Comment:
You should check that the teensy firmware version is ABOVE 0.0.13. If it is, then you can use the new code path.

If not, you can fall back on what you said is obsolete

Reply:
Is there a better way to check this other than just reading the changelog file in teensy-to-any